ResourceStatusSystem/PolicySystem/Actions/SlackAction.py

```diff
@@ -0,0 +1,122 @@
+""" SlackAction
+
+  This action posts the status change to a Slack channel
+  It uses Slack Webhooks.
+  Also compatible with Mattermost, which is an open source alternative to Slack.
+
+  To create a webhook URL refer the following :
+  Slack : https://api.slack.com/incoming-webhooks
```

Comment:
add the syntax to turn these into proper links

Reply:
Direct URLs are automatically linked in Sphinx as it follows reST. 
Should I add a link to the work "Slack" instead?

Reply:
Right, this is fine then.